.angular-cli.json

```diff
@@ -59,6 +59,9 @@
   },
   "defaults": {
     "styleExt": "scss",
-    "component": {}
+    "component": {},
+    "build": {
```

Comment:
Please fix circular dependency issue, 
You just hide the warning not resolved the issue., 
I don't have the words but it is a very bad practice.